client/landing/subscriptions/components/settings-popover/site-settings/site-settings.tsx

```diff
@@ -15,17 +20,28 @@ type SiteSettingsProps = {
 };
 
 const SiteSettings = ( {
+	notifyMeOfNewPosts,
+	onNotifyMeOfNewPostsChange,
+	updatingNotifyMeOfNewPosts,
 	deliveryFrequency,
 	onDeliveryFrequencyChange,
 	onUnsubscribe,
 	unsubscribing,
 	updatingFrequency,
 }: SiteSettingsProps ) => {
 	const translate = useTranslate();
+	const { isLoggedIn } = SubscriptionManager.useIsLoggedIn();
 
 	return (
 		<SettingsPopover>
-			<PopoverMenuItem itemComponent="div">
+			{ isLoggedIn && (
+				<NotifyMeOfNewPostsToggle
+					value={ notifyMeOfNewPosts }
+					onChange={ onNotifyMeOfNewPostsChange }
+					isUpdating={ updatingNotifyMeOfNewPosts }
+				/>
+			) }
+			<PopoverMenuItem itemComponent="div" className="settings-popover__delivery-frequency-item">
```

Comment:
It would look better if we moved `The PopoverMenuItem` component inside `DeliveryFrequencyInput`, but I didn't want to add more changes to the PR.